parts/linux/cloud-init/artifacts/cse_install.sh

```diff
@@ -379,6 +379,10 @@ downloadAzureCNI() {
         echo "VNET_CNI_PLUGINS_URL is not set. Exiting..."
         return
     fi
+
+    if [ $PACKAGE_DOWNLOAD_BASE_URL == "packages.aks.azure.com" ]; then
+        VNET_CNI_PLUGINS_URL="${VNET_CNI_PLUGINS_URL//"acs-mirror.azureedge.net"/$PACKAGE_DOWNLOAD_BASE_URL}"
```

Comment:
Can we add a log_to_event with a pattern for tracing the URL used to download the package